app/scripts/views/mixins/resend-mixin.js

```diff
@@ -43,8 +43,8 @@ define([
 
     _updateResendButton: function () {
       var self = this;
-      // Hide the button after 4 attempts. Redisplay button after a delay.
-      if (self._attemptedSubmits === TOO_MANY_ATTEMPTS) {
+      // Hide the button after too many attempts. Redisplay button after a delay.
+      if (self._attemptedSubmits >= Constants.EMAIL_RESEND_MAX_TRIES) {
         self.logScreenEvent('too_many_attempts');
```

Comment:
I'm leaning toward a stateless fxa-client (seems like it should only be concerned with throttling responses that come from the auth-server anyway) and probably moving it to some third place, i.e. a resend model that could be instantiated for each action that should be limited.